packages/hash/shared/src/blockPaths.json

```diff
@@ -1,11 +1,11 @@
 {
-  "https://block.blockprotocol.org/paragraph": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/paragraph",
-  "https://block.blockprotocol.org/header": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/header",
-  "https://block.blockprotocol.org/person": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/person",
-  "https://block.blockprotocol.org/image": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/image",
-  "https://block.blockprotocol.org/table": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/table",
-  "https://block.blockprotocol.org/divider": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/divider",
-  "https://block.blockprotocol.org/embed": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/embed",
-  "https://block.blockprotocol.org/code": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/code",
-  "https://block.blockprotocol.org/video": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/video"
+  "https://blockprotocol.org/blocks/@hash/paragraph": "https://blockprotocol.org/blocks/@hash/paragraph",
```

Comment:
Hmmm This mapping is essentially an identify function now. Do we need to keep it? I understand that the file can be useful for locally developed blocks, but maybe we can only use it for the overrides? I.e. we'll have `blockPathLookup[key] ?? key` somewhere.

Reply:
I do believe this would more or less be equivalent to a list of default blocks. Nate and Ciaran both commented on the file above, but I am not sure what the impact of removing this would be. I assume it could be removed as well

Reply:
> but maybe we can only use it for the overrides? I.e. we'll have blockPathLookup[key] ?? key somewhere.

This is also what's done currently below, you're right it doesn't do much currently. Searching for usages of `blockpaths.json` in the code i only find this location
https://github.com/hashintel/hash/blob/f485c840574255ecf0798ec98af9dc3896a231c6/packages/hash/shared/src/blockMeta.ts#L55